File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
i, j, k = map(int, input().split())

if (i == j and j == k):
    print(10000+i*1000)

elif (i == j or j == k):
    print(1000+j*100)

elif (i == k):
    print(1000+i*100)
else:
    print(max(i, j, k) * 100)

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.cs
string[] user = Console.ReadLine().Split();
int a = int.Parse(user[0]);
int b = int.Parse(user[1]);
int c = int.Parse(user[2]);

if (a == b && b == c)
{
    Console.WriteLine(10000 + a * 1000);
}
else if (a == b || a == c)
{
    Console.WriteLine(1000 + a * 100);
}
else if (b == c)
{
    Console.WriteLine(1000 + b * 100);
}
else 
{
    Console.WriteLine(Math.Max(a, Math.Max(b, c))*100);
}